pkg/pillar/types/global.go

```diff
@@ -122,6 +122,9 @@ const (
 	MetricInterval GlobalSettingKey = "timer.metric.interval"
 	// DiskScanMetricInterval global setting key
 	DiskScanMetricInterval GlobalSettingKey = "timer.metric.diskscan.interval"
+	// CertExpireInfo and CertExpireWarn global thresholds (in days)
```

Comment:
The code seems to use seconds not days (the default value is e.g., 60*24*HourInSec)